src/test/java/stepDefinition/Rooms_Are_Displayed.java

```diff
@@ -0,0 +1,116 @@
+package stepDefinition;
+
+import Elements.RoomDisplayed_Elements;
+import cucumber.TestContext;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import managers.PageObjectManager;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import utils.ConfigFileReader;
+
+import java.time.Duration;
+import java.util.ArrayList;
+
+import static java.time.Duration.ofSeconds;
+
+public class Rooms_Are_Displayed {
+    TestContext testContext;
+    WebDriver driver;
+    ConfigFileReader configReader;
+    PageObjectManager pageObjectManager;
+
+
+    public Rooms_Are_Displayed(TestContext context) {
+        testContext = context;
+        driver = testContext.getWebDriverManager().getDriver();
+        configReader = testContext.getConfigFileReader();
+        pageObjectManager = testContext.getPageObjectManager();
+
+    }
+
+    @Given("The user is at the home page")
+    public void the_user_has_logged_in_the_website() {
+        driver.get("https://dev-aluvegh.co.za/index.html");
+    }
+
+    @When("The user clicks on the {string} top menu")
+    public void theUserClicksOnTheTopColumn(String column) {
+        WebElement roomColumn = driver.findElement(RoomDisplayed_Elements.topColumn(column));
+        roomColumn.click();
+    }
+
+    @And("The user clicks on the Click to Select Dates tab")
+    public void theUserClicksOnTheClickToSelectDatesTab() {
+        WebElement dateSelect = driver.findElement(RoomDisplayed_Elements.inputDate);
+        dateSelect.click();
+    }
+
+    @Then("User validates calendar is present")
+    public void userValidatesCalendarIsPresent() {
+        WebElement calendar = driver.findElement(RoomDisplayed_Elements.calendarTable);
+        calendar.isDisplayed();
+    }
+
+    @When("User selects the {string} and {string}")
```

Comment:
the step name is not readable 
User selects the check in date {string} and check out date {string}

Reply:
Done